workflow-templates/check-license.yml

```
@@ -59,13 +59,29 @@ jobs:
 
           echo "result=$RESULT" >> $GITHUB_OUTPUT
 
-  check-license:
+  build:
+    name: ${{ matrix.check-license.path }}
     needs: run-determination
     if: needs.run-determination.outputs.result == 'true'
     runs-on: ubuntu-latest
     permissions:
       contents: read
 
+    strategy:
+      fail-fast: false
+
+      matrix:
+        check-license:
+          #TODO Add additional paths where license files should be
+          - path: ./
+            EXPECTED_LICENSE_FILENAME: LICENSE.txt
+            EXPECTED_LICENSE_TYPE: AGPL-3.0
```

```suggestion
            expected-filename: LICENSE.txt
            expected-type: AGPL-3.0
```

Use the standardized "kebab-case" style for key names in this context. GitHub established the convention of using "SCREAMING_SNAKE_CASE" for the key names in the `env` context, but that is the only context where that style is used.

---

Got it:
- `SCREAMING_SNAKE_CASE` for env context 😱 🐍 
- `kebab-case` for non-env conexts 🍖 😋